src/sections/header.html

```diff
@@ -1,4 +1,22 @@
-<div>
-    <h1 class="text-blue-800">Header</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
-</div>
\ No newline at end of file
+<section>
+    <nav class="flex justify-between p-4 mx-2 my-6 z-10 relative items-center">
+        <div>
+            <p>DE</p>
+            <p>EN</p>
+        </div>
+        <div class="self-center w-40">
+            <img src="/assets/img/Baumeister.svg" alt="">
+        </div>
+
+        <!-- Icon -->
+        <img class="cursor-pointer" id="icon" src="assets/img/menu.svg" alt="">
+    
+    </nav>
+
+    <ul id="menu" class="hidden flex-col gap-14 text-green-900 text-3xl font-bold font-sans items-center absolute inset-0 bg-[#EFECEA] w-full h-full">
+        <li class="mt-44">About</li>
```

Comment:
Avoid the use of arbitrary values. 
```suggestion
    <ul id="menu" class="hidden flex-col gap-14 text-green-900 text-3xl font-bold font-sans items-center absolute inset-0 bg-primary w-full h-full">
        <li class="mt-44">About</li>
```